packages/elements/src/overlay-menu/index.ts

```diff
@@ -1029,7 +1029,7 @@ export class OverlayMenu extends Overlay {
       .label=${label}
       .subLabel=${subLabel}
       .icon=${icon}
-      .value=${value}
+      .value=${ifDefined(value)}
```

Comment:
`ifDefined()` is no-op here as `.value` is a property not attribute.
https://lit.dev/docs/templates/directives/#ifdefined

We can remove it.

Reply:
if `ifDefined` has removed the value that return from item will be `string` 'undefined' instead of `null`